doc/user_manual/postprocessor.tex

```diff
@@ -1473,6 +1473,41 @@ \subsubsection{Interfaced}
 </Simulation>
 \end{lstlisting}
 
+\paragraph{Method: HistorySetDelay}
+
+This Post-Processor allows history sets to add delayed or lagged
+variables.  For example, if there a variable price that is set hourly,
+than new variable called price\_prev\_hour could be set by using a
+delay of -1.  This can be useful for training a ROM or other data
+analysis.
+
+In the \xmlNode{PostProcessor} input block, one or more of the following XML sub-nodes are required:
+
+\begin{itemize}
+\item \xmlNode{delay}, \xmlDesc{empty}, a delay node with the following required parameters:
+  \begin{itemize}
+  \item \xmlAttr{original}, the variable to start with
+  \item \xmlAttr{new}, the new variable to create
+  \item \xmlAttr{steps}, the delay (if negative) or steps into the future (if positive) to use for the new variable (so -1 gives the previous, 1 gives the next)
+  \item \xmlAttr{default}, the value to use for cases where there is no previous or next value.
```

Comment:
please be verbose here... "there is no previous or next value". What do you mean? in terms of perfect match using the step? need clear explanation for the user